R/data_lizette.R

```diff
@@ -0,0 +1,21 @@
+#' @title Two Years In, Turnover In Trump’s Cabinet Is Still Historically High
```

Comment:
When I check the help file for `cabinet-turnover`, I keep getting a few lines repeated. This may be because someone else has made a help file for `cabinet-turnover`. I searched "government websites" and found out that someone is, 
<img width="402" alt="Screen Shot 2019-12-09 at 11 31 13 AM" src="https://user-images.githubusercontent.com/37080456/70454017-c2cf0900-1a77-11ea-906a-5d437f9f99d1.png">

Reply:
How should I proceed?

Reply:
Good catch. It seems there was unfortunately a redundancy in work and we can't have both. Since your data wrangling is more a bit more thorough than @Starryz's, let's do the following:
* Delete the relevant sections in `data_starry.R` and `process_data_set_starry.R`
* Leaving only your own